Cython/Includes/cpython/dict.pxd

```diff
@@ -1,13 +1,40 @@
 from .object cimport PyObject
 
 cdef extern from "Python.h":
+    # On Python 2, PyDict_GetItemWithError is called _PyDict_GetItemWithError
+    """
+    #if PY_MAJOR_VERSION <= 2
```

Comment:
Might be a CPython-only thing. But given that Py2 is legacy anyway, we might as well declare it like this.